app/grants/templates/grants/detail/info.html

```diff
@@ -119,6 +119,17 @@ <h1 class="font-title-xl my-4 font-weight-bold">
             </p>
           </div>
         {% endif %}
+        {% if grant.telegram_id %}
+        <div class="font-body mb-2">
+          <i style="width: 14px;" class="fab fa-telegram mr-2 text-center"></i>
+          <p class="d-inline-block mb-0">
+            <a href="https://t.me/{{ grant.telegram_id }}" target="_blank" rel="" data-toggle="tooltip" data-html="true" data-placement="top" for="telegram_id"
+            title="Creator's telegram ID">
+              {{ grant.telegram_id }}
+            </a>
+          </p>
+         </div>
+        {% endif %}
```

Comment:
Isn't it for the whitelabel actually? cc @thelostone-mc 
I believe the request came from matic and they use the whitelabel form

Reply:
I do know we intend to retire the white label form sooner or later but you are right !
cc @thesachinmittal on which form should this be available ?
/grants/new or /grants/matic/new  or both  ?

Reply:
Oh yeah that is true! Good catch @octavioamu !

@aamustapha - can you add the form input to `app/grants/templates/grants/new-whitelabel.html` as well? Thats another way grants get created but its specifically styled for Matic. Hopefully we can unify these two templates soon... but for now we have to remember to make the changes in both places

Reply:
Sure